packages/search/addon/components/navi-search-result/report.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Component from '@glimmer/component';
+
+export default class Report extends Component {
```

Comment:
It could just be my preference, but I like to give class names more context within their names. We have lots of objects throughout navi that represent a `Report` in one place or another. Could we call this `class ReportSearchResult` instead?

Reply:
I'd rather keep it as report since it already lives under the `navi-search-result` folder. When you use it you will refer that as `<NaviSearchResult::Report />` so there is contenxt.

Reply:
That would still be the case if you change the class name as that is determined by the file name. What is the advantage to keeping the class just named `Report`?